packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Inserter from '../inserter';
+
+const ButtonBlockAppender = function( { rootClientId, className = '' } ) {
+	return (
+		<Inserter
+			rootClientId={ rootClientId }
+			renderToggle={ ( { onToggle, disabled, isOpen } ) => (
+				<Button
+					className={ `${ className } button-block-appender` }
+					onClick={ onToggle }
+					aria-expanded={ isOpen }
+					disabled={ disabled }
+				>
+					<span className="screen-reader-text">{ __( 'Add Block' ) }</span>
+					<Icon icon="insert" />
+				</Button>
```

Comment:
Forgive me if I missed something about this, but why not an `IconButton` with a `label` prop instead of a Button with an icon and a hidden aria text.

Reply:
Not sure, I wonder if it should also use an `aria-label` instead of `screen-reader-text`.

I had an attempt at replacing it with an IconButton, but that component has some high specificity on its hover styles that are hard to override.